code/core/src/component-testing/components/Subnav.stories.tsx

```diff
@@ -1,13 +1,30 @@
+import React from 'react';
+
 import { action } from 'storybook/actions';
+import { ManagerContext } from 'storybook/manager-api';
 
 import { Subnav } from './Subnav';
 
+const managerContext: any = {
+  state: {},
+  api: {
+    getData: () => ({ importPath: 'core/src/component-testing/components/Subnav.stories.tsx' }),
+  },
+};
+
 export default {
   title: 'Subnav',
   component: Subnav,
   parameters: {
     layout: 'fullscreen',
   },
+  decorators: [
+    (Story: any) => (
+      <ManagerContext.Provider value={managerContext}>
+        <Story />
+      </ManagerContext.Provider>
+    ),
+  ],
```

Comment:
Are there bad performance implications because of this?

Can we pass the `api` as a prop instead?
The Context updates a lot, because it contains our app-state.